imports/ui/pages/Community.js

```diff
@@ -71,13 +72,15 @@ const RoomBlock = ({ activeRoom, list, onChange, onAdd, openRightBlock }) => {
   );
 };
 
-const CommunityRightBlock = ({ activeRoom, roomList, onChange, onClose }) => {
+const CommunityRightBlock = ({ activeRoom, roomList, onChange, onClose, translate }) => {
```

Comment:
Let's move `CommunityRightBlock` to separate file. Do not pass `translate` as prop - use HOC instead.